packages/cli/src/commands/graph/tasks/graphOrderTask.ts

```diff
@@ -97,6 +97,7 @@ export function graphOrderTask(
 
       if (outputGraphToConsole) {
         task.output = `Graph order for '${srcPath}':\n\n${orderedFiles
+          .filter((filePath) => extname(filePath) !== '.hbs') // Hide .hbs from output
```

Comment:
Maybe refactor, this could be a file filter function we pass down through to generateDotLanguage,generateJson.

This way there is less options in the printers.

Reply:
Nevemind we shouldn't do that, I don't recommend passing down a function because of the indirection for calling `writeOutput`. Which (I think) will not work with passing data to a worker.